src/main/java/org/example/commands/AddDependencyCommand.java

```diff
@@ -0,0 +1,44 @@
+package org.example.commands;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Scanner;
+import org.example.exceptions.InvalidArgument;
+import org.example.model.Node;
+
+public class AddDependencyCommand implements CommandInterface {
+
+  @Override
+  public void execute(Map<String, Node> nodeDependencies) {
+    Scanner scanner = new Scanner(System.in);
```

Comment:
remove hard dependency on scanner

Reply:
maintaining all the inputs from single class ConsoleInputService